meta-openpli/recipes-connectivity/samba/samba_4.%.bbappend

```diff
@@ -60,10 +60,10 @@ do_install_append() {
 	rm -fR ${D}${bindir}
 	rm -fR ${D}${sysconfdir}/tmpfiles.d
 	rm -fR ${D}${sysconfdir}/sysconfig
+	rm -f ${D}${sysconfdir}/init.d/samba
 	install -d ${D}/var/lib/samba/private
 	install -d ${D}${sysconfdir}/samba
 	install -m 644 ${WORKDIR}/smb.conf ${D}${sysconfdir}/samba
-	install -d ${D}${sysconfdir}/init.d
```

Comment:
Why remove this if that directory is required on the next line?

Reply:
This is bbappend file.
So we also need to check what is done before in bb file.
There, this directory has already been created for the samba init file: https://github.com/openembedded/meta-openembedded/blob/dfbdd28d206a74bf264c2f7ee0f7b3e5af587796/meta-networking/recipes-connectivity/samba/samba_4.6.2.bb#L120
Why try to do this the second time?